polytope_server/common/mongo_client_factory.py

```diff
@@ -0,0 +1,20 @@
+import pymongo
+import urllib.parse
+import typing
```

Comment:
nit: imports should be grouped into std, 3rd party, 1st party

Reply:
[2ace679](https://github.com/MeteoSwiss-APN/polytope-server/pull/8/commits/2ace679f3100330308176fc136fc9707ecd81086)